lib/topological_inventory/orchestrator/metric_scaler/watcher.rb

```diff
@@ -54,28 +72,28 @@ def scale_to_desired_replicas
         private
 
         def metrics
-          @metrics ||= begin
-            require "topological_inventory/orchestrator/fixed_length_array"
```

Comment:
Same as above.

Reply:
@syncrou @lindgrenj6 I don't insist in moving it to the top of the file, it doesn't have impact to anything, I'm just curious why do you think it's better than all requires on the top of the file. 
I feel this as something like a rubocop issue (although it obviously isn't in our rubocop rules)

Reply:
@slemrmartin maybe I'm lost in some of the history here, but it appears originally it was being loaded at run time.

If we can load at run time vs all at once, there is potential for performance increases.  

I wasn't 100% saying that we should always load at run time, but if they were originally loaded at runtime ( which isn't bad ) then I didn't see a reason to potentially slow down the performance and simply load them all at once. )

Reply:
@syncrou It's just my guess but "require" at the top of the file can slow down the startup time, but speed up runtime, is it correct? 
Anyway it'll be important only when requiring very big files so this is a theoretical discussion